shardingsphere-agent/shardingsphere-agent-plugins/shardingsphere-agent-plugin-metrics/shardingsphere-agent-metrics-api/src/main/resources/interceptors.yaml

```diff
@@ -0,0 +1,52 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+interceptors:
+  - target: org.apache.shardingsphere.proxy.frontend.command.CommandExecutorTask
+    instanceAdvice: org.apache.shardingsphere.agent.metrics.api.advice.CommandExecutorTaskAdvice
+    points:
+      - type: instance
+        name: run
+      - type: instance
+        name: processException
+  - target: org.apache.shardingsphere.proxy.frontend.netty.FrontendChannelInboundHandler
+    instanceAdvice: org.apache.shardingsphere.agent.metrics.api.advice.ChannelHandlerAdvice
+    points:
+      - type: instance
+        name: channelActive
+      - type: instance
+        name: channelRead
+      - type: instance
+        name: channelInactive
+  - target: org.apache.shardingsphere.infra.route.engine.SQLRouteEngine
+    instanceAdvice: org.apache.shardingsphere.agent.metrics.api.advice.SQLRouteEngineAdvice
+    points:
+      - type: instance
+        name: route
+  - target: org.apache.shardingsphere.proxy.backend.communication.jdbc.transaction.BackendTransactionManager
+    instanceAdvice: org.apache.shardingsphere.agent.metrics.api.advice.TransactionAdvice
+    points:
+      - type: instance
+        name: commit
+      - type: instance
+        name: rollback
+  - target: org.apache.shardingsphere.infra.config.datasource.JDBCParameterDecoratorHelper
+    staticAdvice: org.apache.shardingsphere.agent.metrics.api.advice.DataSourceAdvice
+    points:
+      - type: static
+        name: decorate
+        
```

Comment:
Generally, the last line is expected to be a blank line, rather than an indented line.

Reply:
Please check other files. Thanks.

Reply:
ok , will do it.